docs/docs/references/roles.mdx

```diff
@@ -30,6 +30,13 @@ accessible by users with organization roles.
 | Use the SQL runner                                     |       ✅      |       ✅         |       ❌        |             ❌              |       ❌        |
 | Manage project access and permissions                  |       ✅      |       ❌         |       ❌        |             ❌              |       ❌        |
 | Delete project                                         |       ✅      |       ❌         |       ❌        |             ❌              |       ❌        |
+| Create project previews                                |       ✅      |       ❗️         |       ❗️        |             ❗️              |       ❌        |
+
+:::info
+
+Project previews created by Project Developers, Project Editors or Project Interactive Viewers will not have spaces/charts/dashboards copied from the source project.
+
+:::
```

Comment:
Suggestions:
- move this warning to the organisation table where we have a "create project" row
- have a checkbox followed by a `*` with a link to the text section